src/components/FileUploader/_file-uploader.scss

```diff
@@ -10,6 +10,16 @@
     @include type-scale-item(a, false);
   }
 
+  .#{$prefix}--label-detail {
```

Comment:
This will eventually get its own component, I'm guessing!

Reply:
Yup! This is sorta like label-tooltip-beta-1.0 ya know?

Reply:
👍